pyproject.toml

```
@@ -1,15 +1,58 @@
 [build-system]
+build-backend = "setuptools.build_meta"
 requires = [
-    "setuptools >= 40.0",
+  "setuptools",
```

Let's keep the min version:
```suggestion
  "setuptools >= 40.0",
```